src/gentropy/common/spark_helpers.py

```diff
@@ -375,6 +376,78 @@ def order_array_of_structs_by_field(column_name: str, field_name: str) -> Column
     )
 
 
+def order_array_of_structs_by_two_fields(
+    array_name: str, col1: str, col2: str
```

Comment:
NIT: Could you specify the ordering for col1 and col2 ? Otherwise the function name IMO is too generic. The docstring explains more what happens tho

Reply:
I have renamed variables + added an example. I think this would help, however the sorting expression extremely complicated. It's not my expertise to write these kind of expression, so if you have ideas how to refactor, that would be great.